js_functions/task4.js

```diff
@@ -0,0 +1,25 @@
+function devide(numerator, denominator) {
+   try {
+    if (denominator === 0) {
+        throw new Error("Division by zero is not allowed")
+    }
+    if (typeof numerator !== 'number' || typeof denominator !== 'number') {
+        throw new Error("Please enter number")
```

Comment:
I would improve error message like "numerator & denominator should be numbers"

Reply:
done